docs/charter.md

```diff
@@ -114,13 +115,12 @@ If you represent the management or legal department at your business and/or need
 
 ## Responsibilities and Mandates
 
-While the _CPANSec_ has no formal mandate or responsibilities, it may still seek and accept public statements of support or acknowledgement from the following relevant community stakeholders.
-In addition, the _CPANSec_ may ask for formal mandates from _delegating authorities_ (DA):
+While the _CPANSec_ has no formal mandate or responsibilities, it may still seek and accept public statements of support, acknowledgement or formal mandates from the following relevant community stakeholders.
```

Comment:
does that mean we must update the charter if we ever get a formal mandate to do something?

Reply:
One of the things here I wonder is if anyone ever received a formal mandate for anything.  I suppose Larry may have approved some things but I get the feeling many things happened because some one showed up to do it.

Reply:
I guess it depends on one's definition of "formal". I'm ok with defining formal as "the current project's leadership placed their intent in writing", either via email or meetup or chat or whatever. So if, say, the PSC asks us in writing to handle X, it would be a "formal mandate".

Is that what you meant, @sjn ?

Reply:
For me, a "formal mandate" is achieved when…

* One organization publicly gives a responsibility/mandate to another organization,
* …and the receiving organization publicly accepts this responsibility/mandate,
    * …with the implication that this responsibility/mandate _can be withdrawn_ by the organization giving it, if they deem it necessary.

For me, "publicly give" or "publicly accept" implies some sort of public record being used, e.g. a weekly report mail, or some meeting minutes where a vote was successfully conducted on the matter.

Reply:
@sjn should we have to update our charter if we ever receive formal mandates?